src/main/java/org/sagebionetworks/bridge/config/SpringConfig.java

```diff
@@ -50,6 +50,7 @@
 import org.hibernate.engine.spi.SessionFactoryImplementor;
 import org.hibernate.event.service.spi.EventListenerRegistry;
 import org.hibernate.service.spi.ServiceRegistryImplementor;
+import org.sagebionetworks.bridge.dynamodb.*;
```

Comment:
I would avoid this. Both Eclipse and IntelliJ will just expand this again, causing a lot of churn in PRs.

Reply:
Thank you! I didn't even know that I did this and will fix it now. Also thank you so much for all these comments!! I'm going through and addressing them all now.

Reply:
Additional clarification: There's a project setting in IntelliJ to automatically collapse imports with *. It defaults to something low like 3. I bumped mine up to 999, which basically means it'll never collapse them.

The import sort order we decided on is (iirc)
* import static
* (blank line)
* import java.*
* import javax.*
* (blank line)
* import everything else
* (blank line)
* import org.sagebionetworks.bridge.*

I can send you my code style config, so hopefully your formatting will be the same as ours.